app/controllers/owner_cpf_validations_controller.rb

```diff
@@ -0,0 +1,25 @@
+class OwnerCpfValidationsController < ApplicationController
```

Comment:
Este controller está bem atípico né?! O nome indica que é uma validação, o `create` não cria nada.... 🤔 


Pensando aqui: pq nao levar esta lógica para uma validação customizada dentro do model de PropertyOwner? Vocês podem receber o CPF como um input normal do formulário de sign up e tratar numa validação personalizada a verificação com a API externa.

Reply:
Você sugere um único formulário dentro da view do PropertyOwner do Devise? E fazer a checagem em cima desse formulário?

Realmente se for isso seria bem mais simples a validação, não seria necessário esse uso atípico ( gostei do adjetivo ) de session e flash ( mas gostei tanto dessa solução achei tão chique  :laughing: ). Porém, pela nossa interpretação do PDF ele meio que pede que o CPF seja validado antes de poder prosseguir com o cadastro do proprietário, daí pra evitar um uso de javascript da vida a gente fez assim.

Já em relação ao create e talz a gente realmente ficou em dúvida na hora de nomear, podemos discutir em dupla amanhã e ver como vamos alterar isso.

Desde já agradeço!!

(Desculpa pelo textão é que fiquei brainstorming aqui a sugestão :pleading_face:)

Reply:
Vocês não precisam interpretar o PDF "ao pé da letra". Quando está sendo pedido que o CPF seja validado antes de prosseguir com o cadastro, significa que o usuário não pode ser cadastrado em banco de dados sem que o CPF seja validado. Por isso, a validação pode ser feita pelo próprio model, como qualquer outra validação de campos de models.

Reply:
Estou 100% com o @akaninja nessa ;)

Reply:
Ahh ok! Dessa forma realmente fica mais simples o código sem precisar dessa gambiarra das sessões, vou parear com o Nathan agora e a gente resolve isso :)